cpp/inline/mcts/SearchThread.inl

```diff
@@ -113,6 +113,16 @@ inline void SearchThread<Game>::init_node(StateHistory* history, node_pool_index
       }
       pseudo_local_vars_.request_items.clear();
     }
+
+    if (node->stable_data().is_chance_node) {
+      for (int i = 0; i < node->stable_data().num_valid_actions; i++) {
+        edge_t* edge = node->get_edge(i);
+        core::action_t action = edge->action;
+        typename Game::Types::ChanceDistribution chance_dist =
```

Comment:
I hate seeing the `typename` keyword unnecessarily. Adding a `using` statement to `SearchThread.hpp` so that you can replace this line with,

```
ChanceDistribution chance_dist = ...;
```